### PR TITLE
add suggestions

### PR DESCRIPTION
**Pull request by getlarge**
My 2 cents, based on what I understand of the plugin. 
I still have to give it a spin!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,39 @@ This repository maintains the following example projects and plugins to showcase
...
+
+- in `npm-install` executor:
+
+  - make buildTarget configurable in the executor options, default to 'build'
+  - use getPackageManagerCommand().install instead to be able to support yarn installation as well
+  - use detectPackageManager() and getPackageManagerVersion() to deduce the userconfig path (e.g. .yarnrc, .npmrc, etc.)
+
+- use [createNodesV2](https://nx.dev/nx-api/devkit/documents/CreateNodesV2) instead of [createNodes](https://nx.dev/nx-api/devkit/documents/CreateNodes) in `tooling/build-env/src/plugin/verdaccio-env.plugin.ts`
+
+- in the plugin code, (maybe I got it wrong) it looks like some targets should only be added to the e2e project, but they are added to all projects.
```
Comment (BioPhoton) [inline README.md:97]: will merge the comment and have a look

**Review by BioPhoton — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (BioPhoton) [reply README.md:111]: created #7

Comment (BioPhoton) [reply README.md:111]: created #8

**Review by BioPhoton — APPROVED** [review]

File: projects/utils/src/lib/utils.spec.ts [hunk 2]
```diff
@@ -2,10 +2,7 @@ import { sortUser } from './utils';
 
 describe('sortUser', () => {
   it('should sort a list of users', () => {
-    expect(sortUser([
-      { name: 'Michael' },
-      { name: 'Alice' },
-    ])).toEqual([
+    expect(sortUser([{ name: 'Michael' }, { name: 'Alice' }])).toEqual([
```
Comment (BioPhoton) [inline projects/utils/src/lib/utils.spec.ts:5]: toStrictEqual would be better

**Comment by getlarge** [comment]
Are we merging this one? Or did you port all this code in other PRs?

**Comment by BioPhoton** [comment]
Up dated again the PR after I moved over comments and reduced target config in #32 .
Let's merge this now?

**Comment by getlarge** [comment]
Yes!

**Pull request merged**